taipy/gui/gui.py

```diff
@@ -963,7 +963,7 @@ def __upload_files(self):
                                 part_file_path.unlink()
                     except EnvironmentError as ee:  # pragma: no cover
                         _warn("Cannot group file after chunk upload", ee)
-                        return
+                        return (f"Cannot group file after chunk upload for {file.filename}", 404)
```

Comment:
Why not use the same error message in _warn as well?
I would use an HTTP code 500 instead of 404...